src/dfx/src/config/dfinity.rs

```diff
@@ -26,6 +27,7 @@ const EMPTY_CONFIG_DEFAULTS: ConfigDefaults = ConfigDefaults {
 const EMPTY_CONFIG_DEFAULTS_BITCOIN: ConfigDefaultsBitcoin = ConfigDefaultsBitcoin {
     enabled: false,
     nodes: None,
+    log_level: Level::Info,
```

Comment:
```suggestion
    log_level: Level::default(),
```

Reply:
I don't think this is possible. We're declaring a constant, and we can only use a constant function here. The default method is not a constant function.

Reply:
ah, makes sense